packages/v4/patternfly-docs/content/design-guidelines/components/table/table.md

```diff
@@ -123,10 +123,11 @@ Adding the ability to favorite is possible for any table variation. Users can se
 2. **Favorites column header (optional):** Allows users to sort by favorites.
 
 #### When to use
-**Use a table with favorites when:**
 * Users may want to easily access their most used/viewed items in a table.
 * You have a long list of items and want to favorite items by default for discoverability.
 
+## Spacing
```

Comment:
This is a tough one because I'd say compact vs default are both variations of a table, the rest are just "features" you can add to a table... not sure how we'd change this though so maybe it's fine to leave it as is, but just thinking out loud in case you have ideas.